packages/vite/src/node/build.ts

```diff
@@ -246,15 +246,15 @@ export function resolveBuildOptions(raw?: BuildOptions): ResolvedBuildOptions {
     // Support browserslist
     // "defaults and supports es6-module and supports es6-module-dynamic-import",
     resolved.target = [
-      'es2019',
+      'es2020',
       'edge88',
       'firefox78',
       'chrome87',
       'safari13.1'
```

Comment:
Do we need to bump these as well?

Reply:
https://github.com/vitejs/vite/pull/7644 updates this part.